scripts/publish.rs

```diff
@@ -25,7 +25,7 @@ const CRATES_TO_PUBLISH: &[&str] = &[
     "peepmatic-souper",
     // cranelift
     "cranelift-entity",
-    "cranelift-wasm-types",
+    "wasmtime-types",
```

Comment:
Can you move this down to the wasmtime section?

Reply:
I cannot, this list is topologically sorted.